stdlib/ballerina-builtin/src/main/ballerina/task/Package.md

```diff
@@ -0,0 +1,161 @@
+## Package overview
+
+This package includes functions to manage task timers and task appointments.
+
+### Task timers
+
+Timers execute periodic tasks. The initial execution of a task happens after a specific time period from the task registration, which is denoted using `delay`.  The frequency at which the tasks needs to be executed is defined using the `interval`. The `delay` and `interval` times are defined in milliseconds.
+
```

Comment:
If the `delay` is not specified, the `delay` will be the same as the `interval`.